src/js/protocols/stm32usbdfu.js

```diff
@@ -85,9 +88,7 @@ STM32DFU_protocol.prototype.connect = function (device, hex, options, callback)
     self.verify_hex = [];
 
     // reset progress bar to initial state
-    TABS.firmware_flasher.flashingMessage(null, TABS.firmware_flasher.FLASH_MESSAGE_TYPES.NEUTRAL)
-                         .flashProgress(0);
-
+    TABS.firmware_flasher.flashingMessage(null, TABS.firmware_flasher.FLASH_MESSAGE_TYPES.NEUTRAL).flashProgress(0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['TABS' is not defined.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5626707)